.php_cs.dist

```diff
@@ -0,0 +1,60 @@
+<?php
```

Comment:
Please make a link to `project/.php_cs.dist` or require it we should have the same configuration.

Reply:
Please use require to the project configuration.